### ISSUE TITLE
Question: Support for chunked transfer encoding & JSON streaming

### ISSUE THREAD

**Issue by danielgerlag** [dapr/dapr#6987]
## In what area(s)?

> /area runtime

Does the service to service invocation support [Chunked transfer encoding](https://en.wikipedia.org/wiki/Chunked_transfer_encoding) if the other service responds in this format?

Follow up, does it support any [JSON streaming ](https://en.wikipedia.org/wiki/JSON_streaming) protocols to facilitate large data transfers?

**Comment by olitomlinson** [comment]
I understand progress is being made and there is partial support.

https://docs.dapr.io/operations/support/support-preview-features/#streaming-for-http-service-invocation

@ItalyPaleAle whats the latest here? Any pertinent changes coming in 1.12? Thanks :)

**Comment by ItalyPaleAle** [comment]
Yes this should work with 1.12! It should be enabled by default and complete

**Comment by olitomlinson** [comment]
@ItalyPaleAle does that mean there needs to be a docs change to something like this ?

![image](https://github.com/dapr/dapr/assets/4224880/d3266ffa-6006-4a32-bb5d-602669f2b90f)
